be/src/runtime/plan_fragment_executor.cpp

```diff
@@ -81,6 +81,7 @@ Status PlanFragmentExecutor::prepare(const TExecPlanFragmentParams& request,
 
     RETURN_IF_ERROR(_runtime_state->init_mem_trackers(_query_id));
     _runtime_state->set_be_number(request.backend_num);
+    _runtime_state->set_backend_id(request.backend_id);
```

Comment:
Need to check if `backend_Id` is set in `request`, for forward compatibility. Or set a default value for `backend_id` in proto file.

Reply:
ok. Now the default value (-1) is set in runtime_state.h. 

If it finds that backend_id equals to -1 during fragment execution, the node statistics of this fragment will not be collected.